recipes/theme-darcula/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "theme-darcula" %}
+{% set version = "3.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3f00a02abc5cb89ef657c499dd500ffa601e7074a873798486da14a27e714832
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - jupyter-packaging >=0.7.9,<0.8
+    - jupyterlab >=3.0.0,<4
+    - nodejs
+    - python >=3.6
+    - pip
+    - setuptools >=40.8.0
+  run:
+    - jupyterlab >=3.0.0,<4
+    - python >=3.6
+
+test:
+  commands:
```

Comment:
there are two `commands` keys here? That smells like invalid yaml :)

Reply:
Surprised it passed both the linter and parsing! I removed the dummy key.